esdoc.json

```diff
@@ -24,6 +24,8 @@
     "asset": "./doc/asset",
```

Comment:
You should add `"tutorial": [ "./doc/tutorial/basics.md" ],` here, and add all those md files.